source/developers-guide/shopware-5-plugin-update-guide/index.md

```diff
@@ -13,83 +13,129 @@ menu_order: 100
-        <div class="example--own-topseller">
-            {block name="frontend_detail_example_headline"}
-                <h1 class="own-topseller--headline">My topseller</h1>
-            {/block}
-
-            {block name="frontend_detail_example_img"}
-                <img class="own-topseller--img" src="{link file={$mediaSelection}}" alt="Test" />
-            {/block}
-
-            {block name="frontend_detail_example_topseller"}
-                <div class="own-topseller--container">
-                    {action module=widgets controller=listing action=top_seller sCategory=3}
-                </div>
-            {/block}
-        </div>
-    {/block}
-{/block}
+    <services>
+
+        <service id="swag_update_plugin.subscriber.template_registration"
+                 class="SwagUpdatePlugin\Subscriber\TemplateRegistration">
+            <argument type="service" id="template"/>
+            <argument>%swag_update_plugin.plugin_dir%</argument>
+            <tag name="shopware.event_subscriber"/>
+        </service>
 
+    </services>
+</container>
 ```
+### 2. Create the template
+The goal is to make this plugin compatible with the new Shopware 5 templates. For this purpose, the following should be considered:
 
-The goal is to make this plugin compatible with both the new Shopware 5 and the old Shopware 4 templates. For this purpose, the following should be considered:
+    Inside the PostDispatch event, we have  load the corresponding template files.
+    The extendsTemplate function should not be used in the new template, otherwise the plugin template cannot be overwritten by other templates.
 
-- Inside the __PostDispatch__ event, we have to distinguish between the different template version and load the corresponding template files.
-- The __extendsTemplate__ function should not be used in the new template, otherwise the plugin template cannot be overwritten by other templates.
-- In order for the plugin template to be easily extended by others, the template adjustments should be extracted into a separate file.
+    In order for the plugin template to be easily extended by others, the template adjustments should be extracted into a separate file.
 
 First, the template structure is revised. The example1.tpl file is now divided into three new files:
 
-- SwagExample1/Views/emotion/detail/example1.tpl (Entry point to extends the emotion template)
-- SwagExample1/Views/responsive/frontend/detail/index.tpl (Entry point to extends the responsive template)
-- SwagExample1/Views/common/frontend/swag_example1/detail_extension.tpl (Contains the source code for the extension)
+    *SwagExample1/Views/emotion/detail/example1.tpl (Entry point to extends the emotion template)
+    *SwagExample1/Views/responsive/frontend/detail/index.tpl (Entry point to extends the responsive template)
+    *SwagExample1/Views/common/frontend/swag_example1/detail_extension.tpl (Contains the source code for the extension)
```

Comment:
this path is also wrong. we do not use a common directory

write the directory you use below: frontend/swag_update_plugin/detail/example1.tpl